### ISSUE TITLE
Rescue service

### ISSUE THREAD

**Issue by smashwilson** [bluek1te/feed-txstate#38]
Develop an Angular service to implement the [Food Rescue FRD](https://docs.google.com/document/d/1RruuxQSvXse99OtAVvjDQ8zJ5ZaLg_TVSvk7iLkVyHY/edit). Comment the hell out of it so it can act as an instructional template for structuring the others. Pair with @BenjaminWalkerBond if we can arrange a time.

**Comment by smashwilson** [comment]
@BenjaminWalkerBond will this depend on your work in #37? I see work in the Rescue components in the diff 🤔

**Comment by BenjaminWalkerBond** [comment]
Yes, it’s linked, but I’ve been mainly testing to see if I can create a rescue service that works. I’d like if you could implement the rescue service the correct way. Feel free to use my code as starter template and change it from there. The biggest thing is making sure it’s clearly commented so that all the developers understand what’s going on behind the scenes! 

I will ask Phil if the food and produce temperature page will stay the same or not tomorrow. 

Also, I would work off of bens_model_creation branch as it has the newest database design.
